2025/build-tooling/demo/final/src/App.tsx

```diff
@@ -20,6 +20,8 @@ import Collection from "@arcgis/core/core/Collection";
 import { webMercatorToGeographic } from "@arcgis/core/geometry/support/webMercatorUtils";
 import { findPlacesNearPoint } from "@esri/arcgis-rest-places";
 import { Nil, Result, ServiceInfo } from "./interfaces";
```

Comment:
To keep types simpler would it make sense to factor the nil type to be just = undefined | null?